localization/src/main/kotlin/edu/artic/localization/BaseTranslation.kt

```diff
@@ -0,0 +1,33 @@
+package edu.artic.localization;
+
+import android.content.Context
+import android.content.res.Configuration
+import java.util.*
+
+/**
+ * Common interface for DAO models which can be treated like translations of some sort of media.
+ *
+ * This must be an `interface` and not a proper superclass due
+ * to [limitations in Moshi][com.squareup.moshi.JsonClass].
+ *
+ * @author Philip Cohn-Cort (Fuzz)
+ */
+interface BaseTranslation {
+
+    fun underlyingLanguage() : String?
```

Comment:
use a `val` here. Anything that does not change across computations can just be a `val`.